x-pack/metricbeat/module/oracle/_meta/docs.asciidoc

```diff
@@ -0,0 +1,29 @@
+This is the https://www.oracle.com[Oracle] module for Metricbeat. It is an alpha release and under active development with feedback from the community. A single Metricset for Tablespace monitoring is added so the community can start gathering metrics from their nodes and contributing to the module.
```

Comment:
```suggestion
This is the https://www.oracle.com[Oracle] module for Metricbeat. It is an beta release and under active development with feedback from the community. A single Metricset for Tablespace monitoring is added so the community can start gathering metrics from their nodes and contributing to the module.
```

Reply:
Thanks for this :slightly_smiling_face:  I have actually remove any mention to the state of the release. Docs like this usually pass under the radar in future updates and might get obsolete without us noticing :smile: